renovate.json5

```diff
@@ -0,0 +1,82 @@
+{
+  extends: [
+    'config:base'
+  ],
+
+  /**
+   * Skip any package file whose path matches one of these. Can be string or glob pattern
+   */
+  includePaths: [
+    'package.json',
```

Comment:
Will this include every package.json across the project or only the one in the root?

Reply:
Only the root one, goot catch, I wanted that for testing but need to expand it to the important ones now.